docs/README.md

```diff
@@ -168,95 +174,95 @@ this program; if not, write to the
 Free Software Foundation, Inc.
 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+
 ## 6.0 Credits
 
 OpenGFX is created by the following people (in reverse alphabetical order):
 
-* Zuu (Leif Linse)
-* Zephyris (Richard Wheeler)
-* Varivar
-* V453000 (Vaclav Benc)
-* uzurpator
-* Spaz O Mataz
-* Soeb (Stanislaw Gackowski)
-* Snail (Jacopo Coletto)
-* skidd13 (Benedikt Brüggemeier)
-* Rubidium (Remko Bijker)
-* Roujin (Manuel Wolf)
-* Red*Star (David Krebs)
-* Raumkraut (Mel Collins)
-* Purno (Mark Leppen)
-* planetmaker (Ingo von Borstel)
-* PikkaBird (David Dallaston)
-* PaulC (Paul Charlesworth)
-* orudge (Owen Rudge)
-* oberhuemer
-* northstar2
-* Mr. X
-* mph (Matthew Haines)
-* molace (Zoltán Molnár)
-* michi_cc (Michael Lutz)
-* mb (Michael Blunck)
-* mart3p
-* Lawton27 (Jack Lawton)
-* LordAzamath (Johannes Madis Aasmäe)
-* lead@inbox (Serge Saphronov)
-* Jonha
-* Irwe (Alexander Irwe)
-* Gwyd
-* Gen.Sniper
-* frosch (Christoph Elsenhans)
-* Froix
-* FooBar (Jasper Vries)
-* erikjanp
-* EdorFaus (Frode Austvik)
-* drginaldee
-* DJ Nekkid (Thomas Mjelva)
-* DanMacK (Dan MacKellar)
-* buttercup
-* bubersson (Petr Mikota)
-* Born Acorn (Chris Jones)
-* Brumi
-* Bilbo
-* BenK
-* Ben_Robbins_ (Ben Robbins)
-* athanasios (Athanasios Arthur Palaiologos)
-* andythenorth (Andrew Parkhouse)
-* AndersI (Anders Isaksson)
-* Ammler (Marcel Gmür)
-* 2006TTD (Anthony Lam)
-
-Translations:
-* Afrikaans:                  telanus
-* Catalan:                    juanjo
-* Chinese (simplified):       siu238X, xiangyigao
-* Chinese (traditional):      siu238X, xiangyigao
-* Croatian:                   Voyager1
-* Czech                       alfergen
-* Dutch:                      Alberth
-* English (US):               Supercheese
-* Finnish:                    juzza1, alluke
-* French:                     arikover
-* German:                     planetmaker, Czeczki
-* Hungarian:                  zaza, Czeczki
-* Indonesian:                 UseYourIllusion
-* Italian:                    Voyager1
-* Japanese:                   HawkEye1015
-* Korean:                     Telk
-* Latin:                      Supercheese
-* Lithuanian:                 Stabilitronas
-* Luxemburgean:               Phreeze
-* Norwegian (bokmal):         Trond, Leifbk
-* Polish:                     TadeuszD
-* Portuguese:                 vesgo
-* Russian:                    George
-* Scotish Gaelic:             GunChleoc
-* Serbian:                    stravagante
-* Slovak:                     Tenebrae
-* Spanish:                    SilverSurveferZzZ
-* Swedish:                    Zuu, Pekkape01
-* Tamil:                      Aswn
-
+| Name              | Realname              | Translations                                  |
```

Comment:
When unifying the contributors and translator tables, the information goes missing that a person can have contributed both (code / graphics) and translations. Just one example where this gets lost is Zuu.

I'd like to keep this information separate as it is really a much different kind of level of involvement. Not sure about whether it's actually used in any way like this: the original idea behind an alphabetical order of languages is to look who is responsible for a translation and contact that person when you speak that language...

Reply:
So having them seperate as before is better in your opinion?

Reply:
I tend do answer 'yes'. Unless you want to add a separate column or two with Graphics and Code... but that doesn't do it justice either. And docs/authorreview.csv is not exactly complete (yet much better than anything, if you're interested in who did what. Integrating that at that level of detail in this commit might be asking a bit too much)